_includes/footer.html

```diff
@@ -1,15 +1,20 @@
 <footer>
   <div class="row">
-	<div class="col-lg-12">
-	  <p class='text-center'>
-	  <a href="https://www.ocf.berkeley.edu">
-		<img src="https://www.ocf.berkeley.edu/hosting-logos/ocf-hosted-penguin.svg"
-	   alt="Hosted by the OCF" style="border: 0;" />
-	  </a>
-	  Copyright &copy; Open Computing Facility, eXperimental Computing Facility 2017.
-	  </p>
-	</div>
-	<!-- /.col-lg-12 -->
+    <div class="col-lg-12 text-center">
+      <p>
+        <a href="https://digitalocean.com">
+          <img src="images/digitalocean.png" style="height: 34px; width: 200px;" /> </a>
+        With great appreciation to DigitalOcean for sponsoring the
+        Introductory Track of the DeCal in Spring 2018 </p>
```

Comment:
Maybe just say Beginner track to be consistent with our current Beginner/Advanced nomenclature

Reply:
Agreed with @Baisang, also I think the closing tag should go on its own line to make it clear where the tags actually close, but that's a bit of a nitpick.